src/pages/search.js

```diff
@@ -44,13 +44,19 @@ const SearchPage = ({ data, location }) => {
                       className="padding-bottom-5 margin-top-4 usa-prose border-bottom-05 border-base-lightest"
                     >
                       <b className="title">
-                        <a href={r.url}>{r.title}</a>
+                        <a href={r.url} 
+                          dangerouslySetInnerHTML={{
+                            __html: r.title
+                              .replace(/\uE000/g, '<span class="bg-yellow">')
+                              .replace(/\uE001/g, '</span>'),
+                          }}
+                        >
+                        </a>
```

Comment:
Looking through these changes, I got to thinking about whether we might want to add a URL display to this, like we have on the hosted serp, so I went looking at the full /src/pages/search.js file, I see that on [line 13 here](https://github.com/18F/federalist-uswds-gatsby/blob/ceb5469fa71898e9f6a87e73fa0b6a02239f244f/src/pages/search.js#L13) they're calling `{endpoint}/api/v2/search/i14y` - and that endpoint is our basic [`search.usa.gov`](https://github.com/18F/federalist-uswds-gatsby/blob/ceb5469fa71898e9f6a87e73fa0b6a02239f244f/gatsby-config.js#L46). 

We should change the [api call here](https://github.com/18F/federalist-uswds-gatsby/blob/ceb5469fa71898e9f6a87e73fa0b6a02239f244f/src/pages/search.js#L13) to point to the GSA-branded endpoint and path. Should do this for both the gatsby and [jekyll](https://github.com/18F/federalist-uswds-jekyll/blob/f543121a3578b84f72d9235eb43744fb0540d033/search/index.html#L12) templates

To my original point, I think we should pass on adding url display for now, since it's consistent in both environments. Can tackle that later if it comes up as a need. Customers haven't asked for it so far.

Reply:
Updated!